README.md

```diff
@@ -157,6 +157,8 @@ rocketchat:
   outputformat: "all" # all (default), text, fields
   minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
   # messageformat: "Alert : rule *{{ .Rule }}* triggered by user *{{ index .OutputFields \"user.name\" }}*" # a Go template to format Rocketchat Text above Attachment, displayed in addition to the output from `ROCKETCHAT_OUTPUTFORMAT`, see [Slack Message Formatting](#slack-message-formatting) in the README for details. If empty, no Text is displayed before Attachment.
+  # mutualtls: false # if true, checkcert flag will be ignored (server cert will always be checked)
+  # checkcert: true # check if ssl certificate of the output is valid (default: true)
```

Comment:
```suggestion
```

Reply:
Is RocketChat output considered an exclusive SaaS output? I thought you could deploy it on a kubernetes cluster behind an ingresstls object. Maybe the mutualtls flag can be removed, but the checkcert flag too?

Reply:
I've been confused once again by collapse of line, sorry.